pkg/zio/textio/writer.go

```diff
@@ -39,8 +39,16 @@ func (t *Text) Write(rec *zng.Record) error {
 		for k, col := range rec.Descriptor.Type.Columns {
 			var s, v string
 			if !t.EpochDates && col.Name == "ts" && col.Type == zeek.TypeTime {
```

Comment:
I know this isn't in the scope of this pr but... should we get rid of the `col.Name == "ts"` check and just do this on every `type.TypeTime`?

Reply:
I wondered about that too. I'm not sure what it would do to logs with multiple time fields that are mostly unset....